rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2019-07-10
```

Comment:
Do you need nightly at all?  Why not use stable?

Reply:
I think we don't need it... I choose it because core is nightly too.

Reply:
But I don't quite know what I'm doin' xD